samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -249,6 +249,37 @@ Configure the Sensor Setup Server model on the **Mesh Sensor** node:
 
 The Sensor Setup Server model is now configured and able to receive sensor setting messages from the Sensor Client.
 
+Occupancy Mode
+--------------
+
+You can combine this sample with the :ref:`bluetooth_mesh_sensor_server` sample to trigger the :ref:`Occupancy On <bt_mesh_light_ctrl_srv_on_event>` event on the Light LC Server by the Occupancy Sensor.
+
+To do this, first configure the Light LC Server on the **Mesh Light Fixture** node:
+
+* Bind the model to **Application Key 1**.
+* Set the subscription parameters: Create a dedicated group address.
+
+Prepare the :ref:`bluetooth_mesh_sensor_server` sample:
+
+* Build, run and provision the **Occupancy Sensor** node as described in the sample's documentation.
+
+Configure the Occupancy Sensor Server that is instantiated on the Element 2 of the :ref:`bluetooth_mesh_sensor_server` sample:
+
+* Bind the model to **Application Key 1**.
+* Set the publication parameters:
+
+  * Destination/publish address: Select the same group as the Light LC Server is subscribed to.
+
+When the Light LC Server is not in the **Standby** state, press ``Button 2`` on the **Occupancy Sensor** node to keep the light in the **On** state.
```

Comment:
I think this part will be difficult to understand. I have a suggestion:
```suggestion
Evaluating occupancy inputs when light is not in **Standby** state:

* Open the **Mesh Light Fixture** node in the mobile app.
* Turn ON the light by sending Generic OnOFF Set with ON value to element 2 (<=rephrase as needed). This will bring the light LC Server out of **Standby** state.
* Now, when the Light LC Server is not in the **Standby** state, press ``Button 2`` on the **Occupancy Sensor** node to keep the light in the **On** state. 
* If the Light LC Server enters the **Standby** state, Occupancy sensor inputs will not have any effect because default value of Light LC Occupancy Mode state is 0. 

When using mobile app for iOS, you can change the value of the Light LC Occupancy Mode state to "1", and see how Occupancy Sensor inputs will turn the light ON from the **Standby** state. To do this:
...
...
```